CMakeLists.txt

```diff
@@ -380,6 +380,7 @@ if (FLB_SYSTEM_MACOS)
   include(cmake/macos-setup.cmake)
 endif()
 
+
```

Comment:
Not sure we need this change.

Reply:
@patrick-stephens Referring to the PR or line 383? 😅

Reply:
I mean the empty line lol